MAUI/DataForm/Validation.md

```diff
@@ -0,0 +1,236 @@
+        bool isPropertyValid = DataForm.Validate(new List<string>());
+
+{% endhighlight %}
+{% endtabs %}
+
+If the data form or property is valid, `true` will be returned. Or else `false` will be returned.
+
+N> For validating value, the new value should be committed in data object. So, `DataFormValidationMode` takes higher priority than `DataFormCommitMode`.
+
+### Custom validation through events
+
+You can validate the entire data form using the [ValidateForm] event of the data form.
+
+{% tabs %}
+{% highlight xaml %}
+
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="CommitMode.MainPage">
+
+    <DataForm:SfDataForm
+        x:Name="DataForm"
+        ValidateForm="DataForm_ValidateForm">
+    </DataForm:SfDataForm>
+
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.DataForm.ValidateForm += DataForm_ValidateForm;
+
+    private void DataForm_ValidateForm(object sender, DataFormValidateFormEventArgs e)
+    {
+        if (e.DataObject != null)
+        {
+            if (e.NewValues != null)
+            {
+                this.DataForm.Validate();
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+You can also validate the each property of data form using the [ValidateProperty] event of the data form.
+
+{% tabs %}
+{% highlight xaml %}
+
+<ContentPage xmlns="http://schemas.microsoft.com/dotnet/2021/maui"
+             xmlns:x="http://schemas.microsoft.com/winfx/2009/xaml"
+             xmlns:DataForm="clr-namespace:Syncfusion.Maui.DataForm;assembly=Syncfusion.Maui.DataForm"
+             x:Class="CommitMode.MainPage">
+
+    <DataForm:SfDataForm
+        x:Name="DataForm"
+        ValidateProperty="DataForm_ValidateProperty">
+    </DataForm:SfDataForm>
+
+</ContentPage>
+
+{% endhighlight %}
+{% highlight c# %}
+
+this.DataForm.ValidateProperty += DataForm_ValidateProperty;
+
+    private void DataForm_ValidateProperty(object sender, DataFormValidatePropertyEventArgs e)
+    {
+        if (e.PropertyName == "FirstName")
+        {
+            if (e.NewValue != null && e.NewValue.ToString().Length > 8)
+            {
+                e.IsValid = false;
+                e.ErrorMessage = "Name should not exceed 10 characters";
+            }
+        }
+    }
+
+{% endhighlight %}
+{% endtabs %}
+
+## Valid message
+
+If the value meets the desired criteria, you can show the [validmessage]. As error message, the valid message will also be displayed at the bottom of the editor.
+
+{% tabs %}
+{% highlight c# %}
+
+    private string name;
+
+    [DataFormDisplayOptions(ValidMessage = "Name length is enough")]
+    [StringLength(10, ErrorMessage = "Name should not exceed 10 characters")]
+    public string Name { get; set; }
+
+{% endhighlight %}
+{% endtabs %}
```

Comment:
Mention about, Validattion events in proper and explain all the parameters of validation event arguments